omnigibson/prims/entity_prim.py

```diff
@@ -334,6 +320,20 @@ def _update_joint_limits(self):
             joint.lower_limit = joint.lower_limit * scale_along_axis
             joint.upper_limit = joint.upper_limit * scale_along_axis
 
+    @property
+    def _articulation_view(self):
```

Comment:
should this property be public, and the `articulation_view_direct` simply be the protected version, i.e.: `_articulation_view`? Or do you think that the user should never access this explicitly externally?

Reply:
I don't think the user should ever have explicit access to this, because it skips things like BoundingBoxAPI.clear() if you just use this. That said, the checks here are only important temporarily - I think after we test this for a bit we can safely remove this accessor and just access the field directly.